python/ray/tune/trainable/trainable.py

```diff
@@ -568,25 +599,37 @@ def restore(self, checkpoint_path: str, checkpoint_node_ip: Optional[str] = None
                 f"Got checkpoint path: {checkpoint_path} and IP {checkpoint_node_ip}"
             )
 
-        with open(checkpoint_path + ".tune_metadata", "rb") as f:
-            metadata = pickle.load(f)
+        checkpoint_path = TrainableUtil.find_checkpoint_dir(checkpoint_path)
```

Comment:
should we be consistent with when we say `checkpoint_dir` vs `checkpoint_path`?
with the introduction of `relative_checkpoint_path`, seems we can achieve that?
is it actually checkpoint_dir here?

Reply:
Yes I agree - from this point it's `checkpoint_dir`. Updated